types/internal/deviceSimulator/default.schema.json

```diff
@@ -17,13 +12,40 @@
       "minimum": -3276.5,
       "maximum": 3276.5
     },
+    "humidity": {
+      "title": "Humidity",
+      "unit": "%",
+      "type": "number",
+      "description": "The relative humidity in %",
+      "minimum": 0,
+      "maximum": 100
+    },
     "co2": {
       "title": "CO2",
       "unit": "ppm",
       "type": "integer",
       "description": "Carbon Dioxide in ppm",
       "minimum": 0,
       "maximum": 10000
+    },
+    "noise": {
+      "title": "Noise",
+      "unit": "dBA",
+      "description": "Average value of the sound pressure level in decibel",
+      "type": "number",
+      "minimum": 20,
+      "maximum": 120
+    },
+    "illuminance": {
+      "title": "Light",
+      "type": "number",
+      "unit": "lux",
+      "description": "Light value"
+    },
+    "occupancy": {
```

Comment:
why move away from occupied?

Reply:
Occupancy seems to be used more often than occupied.

Reply:
occupied makes more sense when it comes to the name or not?

Reply:
I would differentiate between state which can take more values than just true/false and purely boolean values